components/library/src/content/ser.rs

```diff
@@ -12,6 +13,9 @@ pub struct TranslatedContent<'a> {
     lang: &'a str,
     permalink: &'a str,
     title: &'a Option<String>,
+    /// The path to the markdown file; useful for retrieving the full page through
+    /// the `get_page` function.
+    file_path: &'a Path,
```

Comment:
it's called `path` in the other context so it should be renamed to `path`.

Reply:
Done!